dm_cli/import_package.py

```diff
@@ -46,9 +75,13 @@ def import_package_tree(package: Package, destination: str) -> None:
     destination_parts = destination.split("/")
     data_source = destination_parts[0]
 
-    documents_to_upload: List[dict] = []
     if len(destination_parts) == 1:  # We're importing a root package
-        documents_to_upload.append(package.to_dict())
+        dmss_api.document_add(
```

Comment:
with this change, it looks like the same documents are updated twice. Once here, and another time inside the IncrementalBar part on line 125. 

Is this the desired behavior?

Reply:
It only uploaded once, since we do it here, instead of appending it to the list of document to be uploaded.